src/store/app/app.selectors.ts

```diff
@@ -76,16 +82,27 @@ export const unsupportedNetworkSelector = createSelector(
   }
 );
 
+export const chainsInCurrentNetworkSelector = createSelector(
+  chainIdSelector,
+  (chainId) => {
+    if (chainId === undefined) return undefined;
```

Comment:
I think you could return empty array here
```suggestion
    if (chainId === undefined) return [];
```
In this approach the conditional render in component won't be necessary